feat: add enterprise-integrated-channels dependency

sameeramin opened the pull request:
This PR adds the `enterprise-integrated-channels` dependency to `edx-platform`. Initially, it won't have any impact and will help us test it against certain tenants available in Production, after which we will gradually migrate our data and jobs channel by channel.

sameeramin committed:
feat: add `enterprise-integrated-channels` dependency

sameeramin committed:
feat: Recompile Python dependencies

Commit generated by workflow `openedx/edx-platform/.github/workflows/compile-python-requirements.yml@refs/heads/master`

sameeramin committed:
Merge pull request #36936 from openedx/sameeramin/compile-python-deps-2a5aed4

chore: Recompile Python dependencies

timmc-edx commented on requirements/edx/kernel.in:
```diff
@@ -72,6 +72,7 @@ edx-codejail>=4.0.0
 edx-django-utils>=5.14.1             # Utilities for cache, monitoring, and plugins
 edx-drf-extensions
 edx-enterprise
+enterprise-integrated-channels      # Integrated Channels to transmit content metadata and learner data to enterprise customers
```
I think we might want to put this in `bundled.in` instead -- can we try that? (From looking at the file comments, I think the idea is that we should put things into that file instead of `kernel.in` as long as that won't cause tests to fail.)

sameeramin replied:
Sure! I can try that.

sameeramin committed:
feat: add `enterprise-integrated-channels` dependency

sameeramin committed:
Merge branch 'master' into sameeramin/ENT-10384

timmc-edx commented:
CI checks failed due to a billing issue that has since been resolved -- you'll probably need to make a change to the PR (e.g. rebasing) in order to get them to re-run. But you may want to wait a bit as there will be a lot of people doing the same thing, and there are concurrency limits.

sameeramin committed:
Merge branch 'master' into sameeramin/ENT-10384

sameeramin committed:
feat: Recompile Python dependencies

Commit generated by workflow `openedx/edx-platform/.github/workflows/compile-python-requirements.yml@refs/heads/master`

sameeramin committed:
Merge pull request #36944 from openedx/sameeramin/compile-python-deps-dd142e5

chore: Recompile Python dependencies

sameeramin committed:
Merge branch 'master' into sameeramin/ENT-10384

sameeramin commented:
@timmc-edx I've updated the PR, and I hope there won't be any CI failures this time.

timmc-edx reviewed (approved):
Looks good!
